TESTS.md

```diff
@@ -0,0 +1,38 @@
+### [<= Back to `README.md`](./README.md)
+## Running Unit Tests
+### Create `token.txt` in `~/Smart-Ledger/Back-End/tests/`
+```shell
+touch tests/token.txt
+```
+
+
+#### In `Smart-Ledger/Front-End/.env.development`, change endpoint:
+```shell
+//.env.development
+REACT_APP_API_ENDPOINT=http://ddns.absolutzero.org:5555/
+```
+
+### Start Front-End:
+```shell
+cd ../Front-End
+npm install (only required on first run)
+npm install react-scripts -g (only required on first run)
+npm start
```

Comment:
Note that you can either do this (spin the FE up individually using NPM), or just spin up the docker-containers together with the API using docker-compose.
Some example commands:
```
docker-compose up -d   # starts up everything in order mongo -> api -> client
docker-compose up -d --build   # rebuilds everything and starts them up like above
docker-compose up -d --build api   # rebuild and re-start just the API
docker-compose up -d --build client   # rebuild and re-start just the Client
docker-compose logs -f api    # tail the logs of the API
docker-compose stop api   # stop just the API
docker-compose start api   # start just the API
````
And so on, you can think of other possible things you can do based on these example commands

Reply:
changed instructions to use `docker-compose up -d --build`